core/parse.ml

```diff
@@ -115,9 +140,12 @@ let reader_of_readline ps1 =
   (accessor_fun, populate_fun)
 
 
-let interactive : Sugartypes.sentence grammar = Parser.interactive
-let program : (Sugartypes.binding list * Sugartypes.phrase option) grammar = Parser.file
-let datatype : Sugartypes.Datatype.with_pos grammar = Parser.just_datatype
+let interactive : Sugartypes.sentence LinksLexer.grammar =
+  Parser.interactive
+let program : (Sugartypes.binding list * Sugartypes.phrase option)
+                LinksLexer.grammar = Parser.file
```

Comment:
I first thought this was a comparison. But `grammar` takes `binding list` and `phrase option` as arguments. I would move `LinksLexer.grammar` up on the same line as its arguments.

Reply:
Done